### PR TITLE
connected /users route and update routes to include comments and titles

### PR DESCRIPTION
**Pull request by hkapk**

### CODE REVIEW

File: views/users.handlebars [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <ol class="user-list">
+{{#each users as |user|}}
+    <li>
+      <article class="users">
+        <div class="user-card">
+            <span class="list-item"><strong>Name:</span>
+          <h4>{{user.first_name}} {{user.last_name}}</h4>
+          <span class="list-item"><strong>Email:</span>
+          <h4>{{user.email}}</h4>
+{{/each}}
```
Comment (kngurley25) [inline views/users.handlebars:12]: do we want to display post title(s) with each user?

File: controllers/index.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ const homeRoutes = require('./home-routes.js');
 const dashboardRoutes = require('./dashboard-routes.js');
 const popularRoutes = require('./popular-routes.js');
 
+
```
Comment (kngurley25) [inline controllers/index.js:7]: I think this index file either requires a user route reference, or the home-routes reference needs to be to the api user routes? maybe it works and I'm wrong

**Review by kngurley25 — APPROVED** [review]

File: views/users.handlebars (hunk 1 above)
Comment (hkapk) [reply views/users.handlebars:12]: I was thinking we could have a page to display all users (first name, last name & email) then a specific user page that displays all of their posts and comments.

File: controllers/index.js (hunk 2 above)
Comment (hkapk) [reply controllers/index.js:7]: I added the /users to the home-routes.js file, I think that was working

**Pull request merged**